recipes/r-simcdm/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set version = '0.1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-simcdm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/simcdm_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/simcdm/simcdm_{{ version }}.tar.gz
+  sha256: 39f0c164e9e6b333dffb9b19a785a30ab56d36bcef9b19004a0e6f3bfeedfde3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - llvm-openmp                  # [osx]
+    - r-base
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo >=0.9.200
+    - libblas
+    - libgomp      # [linux]
+    - liblapack
```

Okay, I'll remove them. Where can I see the linkage analysis? 

```suggestion
    - r-base
    - r-rcpp >=1.0.0
    - r-rcpparmadillo >=0.9.200
    - libblas
    - libgomp      # [linux]
```